ios/TransactionListView.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  TransactionListView.swift
+//  Rainbow
+//
+//  Created by Alexey Kureev on 28/12/2019.
+//
+
+import Foundation
+
+fileprivate struct TransactionSection {
+  var header: Date
+  var transactions: [Transaction]
+}
+
+class TransactionListView: UIView {
+  @objc lazy var onItemPress: RCTBubblingEventBlock = { _ in }
+  @objc lazy var onReceivePress: RCTBubblingEventBlock = { _ in }
+  @objc lazy var onCopyAddressPress: RCTBubblingEventBlock = { _ in }
+  @objc var duration: TimeInterval = 0.15
+  @objc var scaleTo: CGFloat = 0.97
+  @objc var enableHapticFeedback: Bool = true
+  @objc var hapticType: String = "selection"
+  @objc var accountAddress: String? = nil {
+    didSet {
+      header.accountAddress.text = accountAddress
+    }
+  }
+  @objc var transactions: [Transaction] = [] {
+    /// Every time we receive a new set of transactions, regroup by mind_at in the format "MMMM yyyy"
+    /// Then, re-render tableView with the new data
+    didSet {
+      var groups: [Date: [Transaction]] = [:]
+      let calendar = Calendar.current
+      
+      for transaction in transactions {
```

Comment:
What do you think about using a grouping Dictionary and we can apply sorting directly on keys.

```
let groupedTransactions = Dictionary(grouping: transactions) { (element) -> Date in
    return groupByDate(transaction.minedAt)
}
let sortedTransactions = groupedTransactions.keys.sorted()
```

Reply:
Oh, this sorting is a bit tricky. The resulting list should look like this:

**Today**
a
b
c
**Yesterday**
a
b
c
**This month**
a
b
c
**December 2019**
a
b
c

Therefore, if I try to sort it by key, the order will be wrong. I wasn't able to find a more compact solution, unfortunately 😞